<!DOCTYPE html>
<!--[if lt IE 7]>      <html class="no-js lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
<!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
<!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
        <title>Computational Genomics @ PRBB</title>
        <meta name="description" content="Starting point for all the computational genomics groups and research in the PRBB">
        <meta name="viewport" content="width=device-width">

        <link rel="stylesheet" href="css/normalize.min.css">
        <link rel="stylesheet" href="css/main.css">

        <script src="js/vendor/modernizr-2.6.1-respond-1.1.0.min.js"></script>
    </head>
    <body>
        <!--[if lt IE 7]>
            <p class="chromeframe">You are using an outdated browser. <a href="http://browsehappy.com/">Upgrade your browser today</a> or <a href="http://www.google.com/chromeframe/?redirect=true">install Google Chrome Frame</a> to better experience this site.</p>
        <![endif]-->

        <div class="header-container">
            <header class="wrapper clearfix">
                <h1 class="title">Computational Genomics @ PRBB</h1>
            </header>
        </div>

        <div class="main-container">
            <div class="main wrapper clearfix">

                <article>
                    <header>
			<h2>Groups</h2>

<nav>
<ul>
<li><a href='#malba'>Mar Albà</a></li>
<li><a href='#rcastelo'>Robert Castelo</a></li>
<li><a href='#eeyras'>Eduardo Eyras</a></li>
<li><a href='#gfilion'>Guillaume Filion</a></li>
<li><a href='#tgabaldon'>Toni Gabaldón</a></li>
<li><a href='#rguigo'>Roderic Guigó</a></li>
<li><a href='#jhecht'>Jochen Hecht</a></li>
<li><a href='#fkondrashov'>Fyodor Kondrashov</a></li>
<li><a href='#tmarques'>Thomas Marques</a></li>
<li><a href='#mmartirenom'>Marc Martí-Renom</a></li>
<li><a href='#anavarro'>Arcadi Navarro</a></li>
<li><a href='#cnotredame'>Cedric Notredame</a></li>
<li><a href='#sossowski'>Stephan Ossowski</a></li>
<li><a href='#jponomarenko'>Julia Ponomarenko</a></li>
<li><a href='#gtartaglia'>Gian Gaetano Tartaglia</a></li>
</ul>
</nav>
<section><a name='malba'></a>
	<h2><a href='http://evolutionarygenomics.imim.es/'>Mar Albà</a></h2>
	<p class='email'>mar.alba - at - upf.edu</p>
	<figure>
		<img src='img/faces/malba.jpg' alt='Mar Albà'>
	</figure>
	<div class='desc'>
		<p class='place'><a href='http://evolutionarygenomics.imim.es/'>Evolutionary Genomics, IMIM</a></p>
		<p class='text'>Research in our group focuses on the use of comparative genomics and large-scale data analysis to gain an understanding on the evolution of genes and genomes and to predict functional elements. We have a strong computational focus but collaborate with experimental groups in some of our projects.</p>
	</div>
</section>
<section><a name='rcastelo'></a>
	<h2><a href='http://functionalgenomics.upf.edu/'>Robert Castelo</a></h2>
	<p class='email'>robert.castelo - at - upf.edu</p>
	<figure>
		<img src='img/faces/rcastelo.jpg' alt='Robert Castelo'>
	</figure>
	<div class='desc'>
		<p class='place'><a href='http://functionalgenomics.upf.edu/'>Functional Genomics, UPF</a></p>
		<p class='text'>The Functional Genomics Group is focused, broadly speaking, in the development of computational tools for a better understanding of biological mechanisms in the context of knowledge of whole genome structure. More concretely, we investigate ways in which microarray data can be integrated with sequence data in order to find regulatory binding <span class='moretxt'>sites of different kinds and approach the combinatorial control exerted by diverse regulatory mechanisms.</span></p>
	</div>
</section>
<section><a name='eeyras'></a>
	<h2><a href='http://regulatorygenomics.upf.edu/group'>Eduardo Eyras</a></h2>
	<p class='email'>eduardo.eyras - at - upf.edu</p>
	<figure>
		<img src='img/faces/eeyras.jpg' alt='Eduardo Eyras'>
	</figure>
	<div class='desc'>
		<p class='place'><a href='http://regulatorygenomics.upf.edu/group'>Regulatory Genomics, UPF</a></p>
		<p class='text'>Our group focuses on the development of computational predictive models for the study of RNA biology. In particular, we are interested in the role of chromatin and non-coding RNAs in the regulation of splicing, and the relevance of these mechanisms in cancer.</p>
	</div>
</section>
<section><a name='gfilion'></a>
	<h2><a href='http://lab.thegrandlocus.com/'>Guillaume Filion</a></h2>
	<p class='email'>guillaume.filion - at - crg.eu</p>
	<figure>
		<img src='img/faces/gfilion.jpg' alt='Guillaume Filion'>
	</figure>
	<div class='desc'>
		<p class='place'><a href='http://lab.thegrandlocus.com/'>Genome Architecture, CRG</a></p>
		<p class='text'>The 20th century has been the triumph of genetics. However, we fully understand a mere 3% of the human genome - that is, the coding genome. More challenging is to understand how this tiny fraction is orchestrated by the much larger regulatory genome. What does the regulatory genome consist of? How <span class='moretxt'>does it encode information? How is it organized and how does it evolve? Contrary to the coding genome, the information of the regulatory genome is context-dependent. For example, the same promoter can have different levels of activity, the same enhancers can activate one gene or another, depending on available transcription factors and on the local chromatin marks. So the DNA sequence is not enough to understand the function of regulatory sequences. Our research lines focus on the influence of the chromatin context on transcription; and on the co-evolution between the genome and its chromatin context.</span></p>
	</div>
</section>
<section><a name='tgabaldon'></a>
	<h2><a href='http://gabaldonlab.crg.es'>Toni Gabaldón</a></h2>
	<p class='email'>toni.gabaldon - at - crg.eu</p>
	<figure>
		<img src='img/faces/tgabaldon.jpg' alt='Toni Gabaldón'>
	</figure>
	<div class='desc'>
		<p class='place'><a href='http://gabaldonlab.crg.es'>Comparative Genomics, CRG</a></p>
		<p class='text'>The main research interest of our group is to understand the complex relationships between genome sequences and phenotypes and how these two features evolve across species. We generally use large-scale phylogenetics approaches that allow looking at the evolution of genomes from the perspective of all of their genes, and we apply <span class='moretxt'>these analyses to a variety of biological questions related the evolution and function of organelles, pathways, and protein and lncRNA families.</span></p>
	</div>
</section>
<section><a name='rguigo'></a>
	<h2><a href='http://big.crg.cat/bioinformatics_and_genomics'>Roderic Guigó</a></h2>
	<p class='email'>roderic.guigo - at - crg.eu</p>
	<figure>
		<img src='img/faces/rguigo.jpg' alt='Roderic Guigó'>
	</figure>
	<div class='desc'>
		<p class='place'><a href='http://big.crg.cat/bioinformatics_and_genomics'>Bioinformatics and Genomics, CRG</a></p>
		<p class='text'>The overarching theme of the research in our group is the understanding of the information encoded in genomic sequences, and how this information is processed in the pathway leading from DNA to protein sequences. We are specficially interestes in RNA processsing and its genetic and epigenetic regulation.</p>
	</div>
</section>
<section><a name='jhecht'></a>
	<h2><a href='http://www.crg.eu/en/core/programmes-groups/genomics-unit'>Jochen Hecht</a></h2>
	<p class='email'>jochen.hecht - at - crg.eu</p>
	<figure>
		<img src='img/faces/jhecht.jpg' alt='Jochen Hecht'>
	</figure>
	<div class='desc'>
		<p class='place'><a href='http://www.crg.eu/en/core/programmes-groups/genomics-unit'>Genomics Unit, CRG</a></p>
		<p class='text'>The CRG Genomics Core Facility was established in 2008 and currently supports microarray and next generation sequencing technologies. Since its foundation, the CRG has made significant investments in Genomics, including the purchase of several next generation sequencing instruments. Several high-throughput technologies have been implemented to assess differential expression, inter-individual genetic variation, <span class='moretxt'>microRNA discovery, genome sequencing, targeted resequencing, epigenetic profiling and identification of binding sites of DNA or RNA-associated proteins.</span></p>
	</div>
</section>
<section><a name='fkondrashov'></a>
	<h2><a href='http://big.crg.cat/evolutionary_genomics'>Fyodor Kondrashov</a></h2>
	<p class='email'>fyodor.kondrashov - at - crg.eu</p>
	<figure>
		<img src='img/faces/fkondrashov.jpg' alt='Fyodor Kondrashov'>
	</figure>
	<div class='desc'>
		<p class='place'><a href='http://big.crg.cat/evolutionary_genomics'>Evolutionary Genomics, CRG</a></p>
		<p class='text'>We are currently a small laboratory mostly utilizing computational tools to invertigate questions of evolution on the molecular, cellular and organismal levels. At the same time we are actively developing an experimental venue of research, where we hope to be able to apply massive DNA sequencing technologies to ask simple questions <span class='moretxt'>on adaptation and selection in the field. We hope that our experimental efforts will supplement our theoretical and computational venues.</span></p>
	</div>
</section>
<section><a name='tmarques'></a>
	<h2><a href='http://bhusers.upf.edu/tmarques/'>Thomas Marques</a></h2>
	<p class='email'>tomas.marques - at - upf.edu</p>
	<figure>
		<img src='img/faces/tmarques.jpg' alt='Thomas Marques'>
	</figure>
	<div class='desc'>
		<p class='place'><a href='http://bhusers.upf.edu/tmarques/'>Primate Genomics Lab, UPF</a></p>
		<p class='text'>Our main line of research is centered in the discovery of the extent of genomic polymorphism within the great ape species. The goal is to create an integrated view of genome evolution by studying changes in the composition, frequency, size and location at every major branch-point of human divergence from other <span class='moretxt'>primates. The results our lines of research are aimed to assess the rate of genome variation in primate evolution, characterize regional deletions and copy-number expansions as well as determine the patterns of selection acting upon them and whether the diversity of these segments is consistent with other forms of genetic variation among humans and great apes.</span></p>
	</div>
</section>
<section><a name='mmartirenom'></a>
	<h2><a href='http://pasteur.crg.es/portal/page/portal/Internet/02_Research/01_Programmes/04_Gene_Regulation_cont/Structural_Genomics'>Marc Martí-Renom</a></h2>
	<p class='email'>marc.marti-renom - at - crg.eu</p>
	<figure>
		<img src='img/faces/mmartirenom.jpg' alt='Marc Martí-Renom'>
	</figure>
	<div class='desc'>
		<p class='place'><a href='http://pasteur.crg.es/portal/page/portal/Internet/02_Research/01_Programmes/04_Gene_Regulation_cont/Structural_Genomics'>Structural Genomics, CRG</a></p>
		<p class='text'>We are interested in the molecular mechanisms that regulate cell fate. To study such mechanisms, we employ the laws of physics and the rules of evolution to develop and apply computational methods for predicting the 3D structures of macromolecules and their complexes.</p>
	</div>
</section>
<section><a name='anavarro'></a>
	<h2><a href='http://biologiaevolutiva.org/anavarro/'>Arcadi Navarro</a></h2>
	<p class='email'>arcadi.navarro - at - upf.edu</p>
	<figure>
		<img src='img/faces/anavarro.jpg' alt='Arcadi Navarro'>
	</figure>
	<div class='desc'>
		<p class='place'><a href='http://biologiaevolutiva.org/anavarro/'>Evolutionary Genomics Lab, UPF</a></p>
		<p class='text'>Currently, the main research goals of the group focus on to elucidating how evolution, and particularly natural selection, has shaped genome and phenotype diversity in our lineage. To this end, we combine experiments, models and data analysis.</p>
	</div>
</section>
<section><a name='cnotredame'></a>
	<h2><a href='http://big.crg.cat/comparative_bioinformatics'>Cedric Notredame</a></h2>
	<p class='email'>cedric.notredame - at - crg.cat</p>
	<figure>
		<img src='img/faces/cnotredame.jpg' alt='Cedric Notredame'>
	</figure>
	<div class='desc'>
		<p class='place'><a href='http://big.crg.cat/comparative_bioinformatics'>Comparative Bioinformatics, CRG</a></p>
		<p class='text'>The main focus of the group is the development of novel algorithms for the comparison of multiple biological sequences. Multiple comparisons have the advantage of precisely revealing evolutionary traces, thus allowing the identification of functional constraints imposed on the evolution of biological entities. Most comparisons are currently carried out on the <span class='moretxt'>basis of sequence similarity. Our goal is to extend this scope by allowing comparisons based on any relevant biological signal such as sequence homology, structural similarity, genomic structure, functional similarity and more generally any signal that may be identified within biological sequences. Using such heterogeneous signals serves two complementary purposes: (i) producing better models that take advantage of the signal evolutionary resilience, (ii) improving our understanding of the evolutionary processes that lead to the diversification of biological functions. All the applications related to our work are provided to the community through an international network of web servers that can be accessed from http://www.tcoffee.org</span></p>
	</div>
</section>
<section><a name='sossowski'></a>
	<h2><a href='http://pasteur.crg.es/portal/page/portal/Internet/02_Research/01_Programmes/05_Genes_and_Disease/06_Genomic_and_Epigenomic_Variation_in_Disease'>Stephan Ossowski</a></h2>
	<p class='email'>stephan.ossowski - at - crg.eu</p>
	<figure>
		<img src='img/faces/sossowski.jpg' alt='Stephan Ossowski'>
	</figure>
	<div class='desc'>
		<p class='place'><a href='http://pasteur.crg.es/portal/page/portal/Internet/02_Research/01_Programmes/05_Genes_and_Disease/06_Genomic_and_Epigenomic_Variation_in_Disease'>Genomic and Epigenomic Variation in Disease, CRG</a></p>
		<p class='text'>The focus of our group is the analysis of Next Generation Sequencing (NGS) data in order to detect genomic, genic and epigenomic variation related to disease or intolerance to specific treatments. We seek to develop analysis tools for related sequencing applications including genome re-sequencing and assembly, DNA methylation detection, transcriptome analysis <span class='moretxt'>and structural variant prediction. We further aim to incorporate all algorithms into a general analysis framework, allowing for quick adaptation to the challenges of coming sequencing technologies and approaches, e.g. single molecule sequencing, longer read length and further increased throughput. The direct and easy applicability of our framework to sequencing data produced at the CRG or freely available via sequencing read archives is of particular importance for us and we make use of our experience with NGS analysis in several collaborations with experimental groups working on both prokaryotes and eukaryotes.</span></p>
	</div>
</section>
<section><a name='jponomarenko'></a>
	<h2><a href='https://biocore.crg.eu'>Julia Ponomarenko</a></h2>
	<p class='email'>julia.ponomarenko - at - crg.eu</p>
	<figure>
		<img src='img/faces/jponomarenko.jpg' alt='Julia Ponomarenko'>
	</figure>
	<div class='desc'>
		<p class='place'><a href='https://biocore.crg.eu'>Bioinformatics Unit, CRG</a></p>
		<p class='text'>We are the CRG Bioinformatics core facility that provides researchers at CRG-CNAG/PRBB and external organizations with services of consultation, planning NGS and other genomic experiments, NGS data processing, analysis and management, software and database development, bioinformatics training, and access to high-performance computing resources at CRG.</p>
	</div>
</section>
<section><a name='gtartaglia'></a>
	<h2><a href='http://big.crg.cat/gene_function_and_evolution'>Gian Gaetano Tartaglia</a></h2>
	<p class='email'>gian.tartaglia - at - crg.eu</p>
	<figure>
		<img src='img/faces/gtartaglia.jpg' alt='Gian Gaetano Tartaglia'>
	</figure>
	<div class='desc'>
		<p class='place'><a href='http://big.crg.cat/gene_function_and_evolution'>Gene Function and Evolution, CRG</a></p>
		<p class='text'>Our research focuses on gene and proteins, since these are the molecules that enable, regulate and control all the chemical processes on which life depends. In order to function the large majority of RNAs and proteins need to fold into a specific three-dimensional structure. The wide variety of highly specific structures <span class='moretxt'>that results from protein folding, and which serves to bring functional groups into close proximity, has enabled living systems to develop astonishing diversity and selectivity in their underlying chemical processes by using a common set of just twenty building blocks, the amino acids. Most of our research activity focuses on the understanding of regulation of gene expression and on the prediction of the mechanisms of protein folding and aggregation through a combination of in vivo, in vitro and in silico studies. We are currently investigating the conditions under which specific cellular pathways become aberrant and give rise to pathologies. This project, that we call pathosome, involves the in silico prediction of changes in protein-protein and protein-gene interactions that impair cell viability and will have an experimental validation in vivo.</span></p>
	</div>
</section>
                </article>

                <aside class="seminars">
                    <h2><a href="http://www.prbb.org/actualitat">PRBB Agenda</a></h2>
                </aside>

                <aside>
                    <h3>Computational Biology in Barcelona</h3>
                    <p><a href="http://en.wikipedia.org/wiki/Computational_Biology">Computational Biology</a> is thriving in Barcelona. There are excellent  groups at the research institutes (<a href="http://www.irbbarcelona.org">IRB</a>, <a href="http://www.crg.eu">CRG</a>) and Universities (<a href="http://www.upf.edu">UPF</a>, <a href="http://www.ub.edu">UB</a>,  <a href="http://www.uab.cat">UAB</a> and <a href="http://www.upc.cat">UPC</a>). Barcelona hosts the <em>National Center for Genome Analyisis</em> (<a href="http://www.cnag.cat">CNAG</a>), the second largest sequencing center in Europe, and the Barcelona Supercomputing Center (<a href="http://www.bsc.cat">BSC</a>), one of  Europe's top High Performance Computing centers. Computational Biology, with emphasis in <em>Genomics</em>, is particularly strong at the <em>Barcelona Biomedical Research Park</em> (<a href="http://www.prbb.org">PRBB</a>), with many groups among the top leaders in their fields.</p><p>The <em>Barcelona Biomedical Research Park</em> (PRBB) is an agglomeration of public research centres located by the sea, which host over 1,100 people, making it one of the largest biomedical research clusters in Europe. The PRBB hosts state-of-the art core facilites, including ultrasequencing, proteomics, light microscopy and scientific computing. Oportunities exists for researchers in Computational Genomics in all stages of their scientific careers.</p>
			<figure>
				<img src='img/bythesea.jpg' alt='PRBB by the sea'>
				<figcaption>PRBB by the beach. <a href="http://www.flickr.com/photos/okokitsme/7439305300/">Source</a></figcaption>
			</figure>
		     <p><a href="http://www.bcn.cat/en/">Barcelona</a>, the Catalan capital, is one of the largest conurbations in the Mediterranean region. Because of its open and vivid character, it is attracting an increasing number of young and challenge-minded people from all around the world. As a result, there is a blooming environment in science, technology and creation.</p>
                </aside>


            </div> <!-- #main -->
        </div> <!-- #main-container -->

        <div class="footer-container">
            <footer class="wrapper">

                <h3>Computational Genomics @ PRBB - 2012 - 2017</h3>

		<div class="logos">
                	<a href="http://www.prbb.org"><img src="img/logoPRBB.gif" alt="PRBB" ></a>
                	<a href="http://www.imim.cat"><img src="img/imim2012.jpg" alt="IMIM" ></a>
                	<a href="http://www.upf.edu/cexs"><img src="img/pupf.gif" alt="UPF" ></a>
                	<a href="http://www.crg.eu"><img src="img/pcrg.gif" alt="CRG" ></a>
                	<a href="http://www.ibe.upf-csic.es/"><img src="img/logo_ibe.jpg" alt="CRG" ></a>
		</div>
		<p class="extra">Header image from <a href="http://www.barcelonaskyline.com/">BarcelonaSkyline.com</a></p>
            </footer>
        </div>

        <script src="//ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min.js"></script>
        <script>window.jQuery || document.write('<script src="js/vendor/jquery-1.8.2.min.js"><\/script>')</script>

        <script src="js/main.js"></script>

<!-- Piwik -->
<script type="text/javascript">
var pkBaseURL = (("https:" == document.location.protocol) ? "https://stats.biocore.crg.cat/piwik/" : "http://stats.biocore.crg.cat/piwik/");
document.write(unescape("%3Cscript src='" + pkBaseURL + "piwik.js' type='text/javascript'%3E%3C/script%3E"));
</script><script type="text/javascript">
try {
var piwikTracker = Piwik.getTracker(pkBaseURL + "piwik.php", 8);
piwikTracker.trackPageView();
piwikTracker.enableLinkTracking();
} catch( err ) {}
</script><noscript><p><img src="http://stats.biocore.crg.cat/piwik/piwik.php?idsite=8" style="border:0" alt="" /></p></noscript>
<!-- End Piwik Tracking Code -->

    </body>
</html>
